### PR TITLE
[New] Monitor changes to draw status

### PR DESCRIPTION
**Pull request by CalebRas**
## Description

This PR implements `Monitor changes to draw status` in `Maps` category.
URL to README: [Monitor changes to draw status](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-MonitorChangesToDrawStatus/Shared/Samples/Monitor%20changes%20to%20draw%20status)

## Linked Issue(s)

- `swift/issues/5267`

## How To Test

- Pan and zoom on the map to see the map's drawing status update.

## Screenshots

monitor-changes-to-draw-status

### CODE REVIEW

File: Shared/Samples/Monitor changes to draw status/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Monitor changes to draw status
+
+Get the draw status of your map view or scene view to know when all layers in the map or scene have finished drawing.
+
+![Image of Monitor changes to draw status](monitor-changes-to-draw-status.png)
```
Comment (philium) [inline Shared/Samples/Monitor changes to draw status/README.md:5]:
```suggestion
![Image of Monitor changes to draw status sample](monitor-changes-to-draw-status.png)
```

File: Shared/Samples/Monitor changes to draw status/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Use case
+
+You may want to display a loading indicator while layers are loading, which could then be removed on `completed`.
```
Comment (philium) [inline Shared/Samples/Monitor changes to draw status/README.md:9]: > which could then be removed on `completed`.

This seems to be an incomplete thought. What is completed? Instead, you could say something like "when all the layers have finished drawing" or "when the draw status for each layer is `completed`".

File: Shared/Samples/Monitor changes to draw status/MonitorChangesToDrawStatusView.swift [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    
+    var body: some View {
+        MapView(map: map)
+            .onDrawStatusChanged { drawStatus in
+                // Updates the state when the map's draw status changes.
+                mapIsDrawing = drawStatus == .inProgress
+            }
+            .overlay(alignment: .top) {
+                // The drawing status text at the top of the screen.
+                Text(mapIsDrawing ? "Drawing..." : "Drawing completed.")
```
Comment (philium) [inline Shared/Samples/Monitor changes to draw status/MonitorChangesToDrawStatusView.swift:43]:
```suggestion
                Text(mapIsDrawing ? "Drawing…" : "Drawing completed.")
```

**Comment by philium** [comment]
Is this ready for re-review?

**Comment by CalebRas** [comment]
> Is this ready for re-review?

@philium Yes, thank you

**Review by philium — APPROVED** [review]

**Comment by CalebRas** [comment]
@mhdostal Just in case you missed this. Let me know if you can't get to it. Thank you!

**Review by mhdostal — APPROVED** [review]

**Pull request merged**